pq/stm32/setup_kyber.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+mkdir pqm4_kyber512
+mkdir Inc
+cur_dir=$PWD
+cd pqm4_kyber512
+code_dir=$PWD
+cd ../Inc
+inc_dir=$PWD
+
+git clone https://github.com/mupq/pqm4.git /tmp/pqm4
```

Comment:
I like the idea of using `/tmp`, but it might be easier to use a relative local directory to make this easier to adopt to .batch file. Also recommend having the `tmp` be a variable.

Reply:
Yeah, I think I can do that.